portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const funFacts =
+      ["I've built 8 computers so far!", 'I interned at Google last summer.', 'I spent my last summer in New York City.', 'I speak Spanish.'];
```

Comment:
nit: A couple things. Your first entry uses double quotes ("") and the rest use single quotes (''), can you make them consistent? Also this line is a little long, can you put each String on a newline?

Reply:
Done.